workshop/SCHEDULE.md

```diff
@@ -7,34 +7,34 @@ nav_title: Schedule
 
 *Draft schedule subject to changes*
 
-| Time        | Topic                                          |
-|-------------|------------------------------------------------|
+| Time        | Topic                                          | Location |
+|-------------|------------------------------------------------|----------|
 | **Day 1**   | **2020-06-22** <br> [Introduction to R and the Tidyverse](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/README.md)                      
-| 12:00       | Welcome, Introductions and Getting Started  |
-| 12:30       | [Introduction to base R](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/01-intro_to_base_R.nb.html) |
-|             | [Introduction to R Exercise](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/04a-intro_to_R_exercise.Rmd)
-| 3:30 		 | [Consultation session](workshop-structure.md#consultation-sessions) begins |
+| 12:00       | Welcome, Introductions and Getting Started  | Zoom: Main Session |
+| 12:30       | [Introduction to base R](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/01-intro_to_base_R.nb.html) | Zoom: Main Session |
+| 3:00        | Questions and introduction to the exercise: <br> [Introduction to R](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/01b-intro_to_base_R_exercise.Rmd) | Zoom: Breakout Rooms |
```

Comment:
Oh my, I hope Intro to R doesn't take 2.5 hours!

Reply:
Agreed. What should it say instead? I would rather overestimate here.

Reply:
Maybe just push it 30 min? Start the breakout at 2:30? I actually suspect we won't really get started with the instruction until closer to 12:45 or 1, so we could make that period longer instead.